### PR TITLE
[11702] Deleting references to org.json so we use just one json lib

### PR DESCRIPTION
**Pull request by pluraliseseverythings**

### CODE REVIEW

File: grakn-graph/src/test/java/ai/grakn/graph/internal/TxCacheTest.java [hunk 1]
```diff
@@ -134,6 +135,21 @@ public void whenDeletingAnInstanceWithNoRelations_EnsureLogIsEmpty(){
     }
 
     @Test
+    public void whenNoOp_EnsureLogNotEmpty() {
```
Comment (fppt) [inline grakn-graph/src/test/java/ai/grakn/graph/internal/TxCacheTest.java]: Personally I think the title of this test is misleading. The Log is empty, it contains no useful information.

**Comment by fppt** [comment]
!rtg - ai.grakn.test.engine.tasks.manager.TaskManagerTest.whenATaskIsStoppedDuringExecution_ItSavesItsLastCheckpoint

**Comment by pluraliseseverythings** [comment]
!rtg

**Comment by fppt** [comment]
!rtg - jenkins did something dumb and died

**Comment by fppt** [comment]
!rtg - ai.grakn.test.engine.controller.CommitLogControllerTest.whenCommittingGraph_TaskManagerReceivesCountTask

**Pull request merged**